src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/Error.razor

```diff
@@ -0,0 +1,16 @@
+﻿@page "/error"

```

Comment:
Update https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/test/template-baselines.json to reflect this new file.

Reply:
Didn't seem to work

Reply:
The `blazorserver` entry there has 4 entries, you only updated one.